README.md

```diff
@@ -1685,7 +1685,7 @@ to `stream` finishes executing. Streaming does not work at all with Shotgun.
 
 If the optional parameter is set to `keep_open`, it will not call `close` on
 the stream object, allowing you to close it at any later point in the
-execution flow. This only works on evented servers, like Thin and Rainbows.
+execution flow. This only works on evented servers, like Rainbows.
 Other servers will still close the stream:
 
 ```ruby
```

Comment:
I missed it!

I tried to rewrite it;


```diff
- set :server, :thin
+ set :server, :rainbows
```

but it returned error such as `/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1716:in `detect_rack_handler': Server handler (rainbows) not found. (RuntimeError)`

I think [this method](https://github.com/rkmathi/sinatra/blob/607346295924c75a86f12ecbad9430cfc430c032/lib/sinatra/base.rb#L1746-L1755) is expected to be passed argument which is conforms to `Rack::Handler.get()`, but rainbows is not found in [lib/rack/handler](https://github.com/rack/rack/tree/2.2.3/lib/rack/handler).

If you know how to fix this problem, please let me know how to fix it.

Reply:
Ah true. At least `Rack::Handler::Rainbows` must exists even if rack does not provide it by default.
I'm going to re:visit here this weekend. I'm sorry for the delay.

Reply:
In order to use the streaming feature, can we put a minimum configuration for rainbows in README instead of specifying :rainbows in the :server settings?
It should work smoothly, of course.

Reply:
I fixed it.
https://github.com/sinatra/sinatra/pull/1627/commits/c0d4dd74183dd57d56e317def5d0b7433ee617bc#diff-04c6e90faac2675aa89e2176d2eec7d8R1691-R1735